src/db.rs

```diff
@@ -952,18 +1004,18 @@ impl DB {
         DBIterator::new_cf(self, cf_handle, &opts, mode)
     }
 
-    pub fn prefix_iterator_cf(
+    pub fn prefix_iterator_cf<P: AsRef<[u8]>>(
         &self,
         cf_handle: ColumnFamily,
-        prefix: &[u8]
+        prefix: P
```

Comment:
```suggestion
        prefix: P
        prefix: impl AsRef<u8>
```

Reply:
But it's a matter of taste.

Reply:
I avoid it because it doesn't allow you to [explicitly specify type arguments](https://github.com/rust-lang/rust/issues/50950).